### ISSUE TITLE
Incorrect API use in models/textsum/seq2seq_attention_model.py

### ISSUE THREAD

**Issue by scylla** [tensorflow/models#1220]
at line number 230
models/textsum/seq2seq_attention_model.py is incorrectly using the api for tf.nn.sampled_softmax_loss,
the order of arg for labels and inputs is reversed in current code

**Comment by nealwu** [comment]
I believe this should be fixed now via https://github.com/tensorflow/models/pull/1226. If you run into further issues let me know.

**Comment by Jacksonlark** [comment]
I find tf.nn.sampled_softmax_loss source def likes:

def sampled_softmax_loss(weights,
                         biases,
                         labels,
                         inputs,
                         num_sampled,
                         num_classes,
                         num_true=1,
                         sampled_values=None,
                         remove_accidental_hits=True,
                         partition_strategy="mod",
                         name="sampled_softmax_loss")

but models/textsum/seq2seq_attention_model.py(line 230) give parameters likes:

tf.nn.sampled_softmax_loss(w_t, v, inputs, labels,
                                              hps.num_softmax_samples, vsize)

clearly, inputs and labels is reversed and make error #1237

**Comment by nealwu** [comment]
Ahh you're right -- sorry, I misread and thought it was talking about a different model. I have a pull request to fix the issue at https://github.com/tensorflow/models/pull/1241.

**Comment by scylla** [comment]
Sorry i was not able to re-check yesterday, i used the same fix locally seemed to work for me

**Comment by nealwu** [comment]
@scylla can you pull again from master and make sure it works for you? Just merged the pull request above.

**Comment by scylla** [comment]
@nealwu currently i am away from my work machine, will do it tomorrow

**Comment by scylla** [comment]
verified the changes working fine now

**Comment by nealwu** [comment]
Ok great!
